src/sidebars/sidebars.json

```diff
@@ -312,7 +312,21 @@
         "entry": "People",
         "name": "People",
         "items": [
-            "handbook/people/hiring-process",
+     ],
+        "child_entries": ["Hiring process"]
+    },
+    {
+        "entry": "Hiring Process",
+        "name": "Hiring Process",
+        "items": [
+            "handbook/people/hiring-process/hiring-process",
+            "handbook/people/hiring-process/design-hiring",
+            "handbook/people/hiring-process/engineering-hiring",
+            "handbook/people/hiring-process/exec-hiring",
+            "handbook/people/hiring-process/operations-hiring"
```

Comment:
Does this line need a comma? Just perusing to see if there’s anything obvious as to why tests would fail.

Reply:
I have absolutely no idea, never had to add child pages to the side bar. Just tried to copy what other people did and probably messed it up haha

Reply:
A comma here would actually be forbidden per JSON spec 😅